eng/emitter-package.json

```diff
@@ -1,17 +1,17 @@
 {
   "main": "dist/src/index.js",
   "dependencies": {
-    "@azure-tools/typespec-rust": "0.23.1"
+    "@azure-tools/typespec-rust": "0.23.2"
   },
   "devDependencies": {
-    "@azure-tools/typespec-azure-core": "0.60.0",
-    "@azure-tools/typespec-azure-rulesets": "0.60.0",
-    "@azure-tools/typespec-client-generator-core": "0.60.1",
+    "@azure-tools/typespec-azure-core": "0.60",
+    "@azure-tools/typespec-azure-rulesets": "0.60",
+    "@azure-tools/typespec-client-generator-core": ">=0.60.1",
     "@typespec/compiler": "1.4",
     "@typespec/http": "1.4",
     "@typespec/openapi": "1.4",
     "@typespec/rest": "0.74",
     "@typespec/versioning": "0.74",
     "@typespec/xml": "0.74"
   }
-}
+}
```

Comment:
```suggestion
}

```

Reply:
Do you have the EditConfig extension installed? It shouldn't, unless you have some overriding settings in your personal settings.